pkgs/by-name/di/diodon/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  desktop-file-utils,
+  glib,
+  gobject-introspection,
+  gtk3,
+  meson,
+  ninja,
+  pkg-config,
+  vala,
+  wrapGAppsHook3,
+  xvfb-run,
+  libayatana-appindicator,
+  libpeas,
+  libXtst,
+  zeitgeist,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Can you move it to `finalAttrs` pattern?
- #315337

Reply:
Sure. I wasn't aware that issue had been settled, w.r.t. performance impacts and such.